muesli/web/forms.py

```diff
@@ -601,10 +601,17 @@ def __init__(self, request):
                 FormField('name',
                    label='Name', size=100,
                    required=True),
-                FormField('category',
```

Comment:
As I've described above, I don't think adding separate URLs is the way to go here. Please reintroduce the category field and hide and preselect it using the template, based on the GET parameter.